pubspec.yaml

```diff
@@ -130,14 +132,14 @@ dependencies:
   wechat_camera_picker: ^3.8.0
   open_file: ^3.3.2
   mime: ^1.0.4
-  any_link_preview: ^3.0.0
   async: ^2.11.0
   cached_network_image: ^3.2.3
   pull_to_refresh: ^2.0.0
   flutter_image_compress: ^2.0.4
   image_gallery_saver: ^2.0.3
   file_saver: ^0.1.1
   flutter_keyboard_visibility: ^5.4.1
+  string_validator: ^1.0.0
```

Comment:
use for what?

Reply:
It's support check if string is URL 
<img width="587" alt="Screenshot 2023-09-18 at 08 52 33" src="https://github.com/linagora/twake-on-matrix/assets/99852347/c728bada-943c-4454-b87c-a558a48acdbe">

Reply:
why not use `URL.parse`?